temoa/extensions/growth_rates/components/growth_capacity.py

```
@@ -0,0 +1,126 @@
+            \qquad \forall \{r, p, t\} \in \Theta_{\text{limit\_degrowth\_capacity}}
+    """
+
+    regions = geography.gather_group_regions(model, r)
+    techs = technology.gather_group_techs(model, t)
+
+    growth = model.limit_degrowth_capacity if degrowth else model.limit_growth_capacity
+    rate = 1 + value(growth[r, t, op][0])
+    seed = value(growth[r, t, op][1])
+    cap_rpt = model.v_capacity_available_by_period_and_tech
+
+    cap_indices = {(_r, _p, _t) for _r, _p, _t in cap_rpt.keys() if _t in techs and _r in regions}
```

Ruff (0.15.20)

[warning] Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)